### PR TITLE
Feature: Initialize Project with Components

### PR DESCRIPTION
**Pull request by luftedar**
# Feature: Initialize Project with Components

In this update: 

- Initialised React-Redux.
- Added Components.
- Added basic CSS for positioning.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @luftedar 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,76 +1,70 @@
...
+# Getting Started with Create React App
 
-# Project Name
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-> One paragraph statement about the project.
+## Available Scripts
 
-![screenshot](./app_screenshot.png)
+In the project directory, you can run:
```
Comment (mikenath223) [inline README.md]: - [x] Kindly update the readme following [this template](https://github.com/microverseinc/readme-template).

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <h5>Genre</h5>
+        <h2>Title</h2>
+        <p>Author</p>
+        <div>
+          <a href="http://localhost:3000/">Comments</a>
+          <a href="http://localhost:3000/">Remove</a>
+          <a href="http://localhost:3000/">Edit</a>
+        </div>
+      </div>
+      <div className="progress" />
```
Comment (mikenath223) [inline src/components/Book.js]: - [x] The Book component is expected to take the book data as a prop and render the title and the author. Kindly fix it.

**Comment by luftedar** [comment]
### Requested changes are made.

**Review by dasileker — APPROVED** [review]
## **Hi @luftedar,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**